qa/rolling-upgrade/build.gradle

```diff
@@ -67,6 +68,10 @@ BuildParams.bwcVersions.withWireCompatible { bwcVersion, baseName ->
     if (excludeList.isEmpty() == false) {
       systemProperty 'tests.rest.blacklist', excludeList.join(',')
     }
+
+    doLast {
+      indexVersion = baseCluster.map(c -> c.getIndexVersion(0)).get()
```

Comment:
This is the bit I'm hazy on - can a task update a variable outside its registration like this, and will `indexVersion` be able to be read properly by dependent tasks?